Apps/WebClient/src/ClientApp/src/components/timeline/addNoteButton.vue

```diff
@@ -0,0 +1,124 @@
+<script lang="ts">
+import Vue from "vue";
+import { Component, Prop } from "vue-property-decorator";
+import { Action, Getter } from "vuex-class";
+
+import UserPreferenceType from "@/constants/userPreferenceType";
+import EventBus, { EventMessageName } from "@/eventbus";
+import User from "@/models/user";
+import { UserPreference } from "@/models/userPreference";
+import { SERVICE_IDENTIFIER } from "@/plugins/inversify";
+import container from "@/plugins/inversify.container";
+import { ILogger } from "@/services/interfaces";
+
+@Component
+export default class AddNoteButtonComponent extends Vue {
+    @Prop({ default: "lg", required: false }) textBreakpoint!: string;
```

Comment:
This property does not seem to be used outside the component, maybe it should be hardcoded until we need to modify it from the parent

Reply:
Actually, I am thinking that both options and add note should be minimized to just the icon under the same breakpoint; same on the calendar view

Reply:
The Options button has text at and above the "md" breakpoint, but Add Note on the timeline view only has text at and above the "lg" breakpoint because the pagination takes up more room than the filter field.  It would be possible to change the Options button breakpoint to "lg" to match, but that would need UI re-approval.  There would also be the question of whether the calendar view needs to be updated to match - it can display text for both buttons at the "md" breakpoint without issue.  I suggest asking @mk08sh to weigh in.

Reply:
It would be just easier to have both on the same breakpoint for calendar and linear and more consistent between options and add-note

Reply:
Timeline MD
![image](https://user-images.githubusercontent.com/16570293/126009218-2f9b434e-08a6-44c5-a7cc-f08c99781bb1.png)

Calendar View MD
![image](https://user-images.githubusercontent.com/16570293/126009277-2f1af269-1817-4768-b176-cc070b3b9f79.png)

Reply:
Screenshots approved by @mk08sh.